src/views/Create.vue

```diff
@@ -403,41 +367,40 @@ export default {
 		margin-bottom: 24px;
 		margin-left: 56px;
 
-		h2 {
-			margin-bottom: 0; // because the input field has enough padding
-		}
-
-		.form-title,
-		.form-desc,
-		.info-message {
-			width: 100%;
-			padding: 0 16px;
-			border: none;
-		}
 		.form-title {
 			font-size: 28px;
 			font-weight: bold;
 			color: var(--color-main-text);
 			line-height: 34px;
 			min-height: 36px;
 			margin: 32px 0;
-			padding-left: 14px; // align with description (compensate font size diff)
-			padding-bottom: 4px;
-			overflow: hidden;
-			text-overflow: ellipsis;
-			resize: none;
+			width: calc(100% - 56px); // margin of header, needed if screen is < 806px (max-width + margin-left)
+
+			>div {
+				padding-left: 14px; //align with description
+				min-height: unset;
+			}
 		}
-		.form-desc {
+
+		.form-desc,
+		.info-message {
 			font-size: 100%;
-			line-height: 150%;
-			padding-bottom: 20px;
+			min-height: unset;
+			padding: 0px 14px 16px;
+			width: calc(100% - 56px);
+		}
+
+		.form-desc {
+			color: var(--color-text-maxcontrast) !important;
+			line-height: 1.5em;
 			margin: 0px;
-			resize: none;
+
+			&.readonly {
+				padding: 0px 15px 17px; // compensate border of editable content
```

Comment:
Still some moving of the elements between view and edit mode for the form description. If you add the 1px padding to the top at least for me it stays in place 🙂 

```suggestion
				padding: 1px 15px 17px; // compensate border of editable content
```

Reply:
Thank you! Works :)

Reply:
Did this disappear again? I can see a small movement when switching between the modes... This might also be related to the NcRichContenteditable as this component seems to have a thinner border than the other input fields. I created an issue for the lib: https://github.com/nextcloud/nextcloud-vue/issues/3635